src/ray/stats/metric_defs.h

```diff
@@ -0,0 +1,29 @@
+#ifndef RAY_METRIC_DEFS_H_
+#define RAY_METRIC_DEFS_H_
+
+/// Use `RAY_METRIC(name, description, unit)` to define a metric
+/// which parameters are `metric_name`, `description` and `unit`.
+
+static Metric TaskElapse = Metric::MakeHistogram("task_elapse",
+                                                 "the task elapse in raylet",
+                                                 "ms",
+                                                 {0, 25, 50, 75, 100},
+                                                 {NodeAddressKey, CustomKey});
+
+static Metric RedisLatency = Metric::MakeHistogram("redis_latency",
+                                                   "the latency of redis",
+                                                   "ms",
+                                                   {0, 200, 400, 600, 800, 1000},
+                                                   {NodeAddressKey, CustomKey});
+
+
+static Metric TaskCount = Metric::MakeCount("task_count",
+                                            "the task count that the raylet received",
+                                            "pcs");
+
+static Metric WorkerCount = Metric::MakeGauge("worker_count",
+                                            "the worker count",
```

Comment:
Also, is this the number of current workers, or total number of workers started by a raylet? 

It's okay to not measure both for now. We can start with a few basic ones. But let's make them more clear.